start_docker_dev.sh

```diff
@@ -1,5 +1,6 @@
 HSUSER=headerservice
 TAGNAME=4.1.1-5.1.0-salobj_5.10.0-hs_2.2.0
```

Comment:
should this not be removed if you re-declare right below?

Reply:
Yes, I'll clean this one up.